saleor/order/models.py

```diff
@@ -306,6 +306,10 @@ class Order(ModelWithMetadata, ModelWithExternalReference):
     voucher = models.ForeignKey(
```

Comment:
Also we need a discussion here if we want to keep the relation to `voucher` or not